packages/flutter_tools/test/integration.shard/command_output_test.dart

```diff
@@ -138,11 +136,11 @@ void main() {
       '--machine',
     ]);
 
-    final Map<String, Object> versionInfo = json.decode(result.stdout
+    final Map<String, dynamic> versionInfo = json.decode(result.stdout
       .toString()
       .replaceAll('Building flutter tool...', '')
       .replaceAll('Waiting for another flutter command to release the startup lock...', '')
-      .trim()) as Map<String, Object>;
+      .trim()) as Map<String, dynamic>;
```

Comment:
These should be `Object` or `Object?` not `dynamic`.  `dynamic` removes static type checking and shouldn't be used in null safe libraries.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic